server/push/controllers/send_push.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import config from '../../config';
+import webpush from 'web-push';
+
+const serverSubject = 'https://github.com/romandev/absolute';
```

Comment:
Let's move this subject into function and I think this should be uppercase only?
const SERVER_SUBJECT = ...
like this?